.github/workflows/database_auto_updater.yml

```diff
@@ -0,0 +1,30 @@
+name: Python package
+
+on:
+  pull_request:
+    # Will trigger on closed PRs
+    types: [closed]
```

Comment:
I think it should trigger on merged PRs?

Reply:
PR have subflag ``merged`` instead of something like ``closed & merged`` flag. It is defined in the jobs section.
``if: ${{ github.event.pull_request.merged == true }}``
The job will only trigger if this is true, if not it will just skip

Reply:
Ohkay so it will trigger on PR merged?

Reply:
yes